recipe/timezone.patch

```diff
@@ -0,0 +1,10 @@
+--- src\turbodbc_arrow\arrow_result_set.cpp
++++ src\turbodbc_arrow\arrow_result_set.cpp
+@@ -2,6 +2,7 @@
+ 
+ // Somewhere a macro defines BOOL as a constant. This is in conflict with array/type.h
+ #undef BOOL
++#undef timezone
```

Comment:
_b_env\include\pyconfig.h at line 230 contains code block with definition of `timezone` for msvc:

```
/* VS 2015 defines these names with a leading underscore */
#if _MSC_VER >= 1900
#define timezone _timezone
#define daylight _daylight
#define tzname _tzname
#endif
```

It causes compilation error for include\arrow\type.h in following code with used `timezone` as variable and method names:

```
explicit TimestampType(TimeUnit::type unit, const std::string& timezone)
      : FixedWidthType(Type::TIMESTAMP), unit_(unit), timezone_(timezone) {}

  Status Accept(TypeVisitor* visitor) const override;
  std::string ToString() const override;
  static std::string name() { return "timestamp"; }

  TimeUnit::type unit() const { return unit_; }
  const std::string& timezone() const { return timezone_; }
```